pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/OverloadShedder.java

```diff
@@ -103,8 +104,10 @@
                 // make up for at least the minimum throughput to offload
 
                 loadData.getBundleData().entrySet().stream()
-                    .filter(e -> !HEARTBEAT_NAMESPACE_PATTERN.matcher(e.getKey()).matches()
-                            && !HEARTBEAT_NAMESPACE_PATTERN_V2.matcher(e.getKey()).matches()
+                    .filter(e -> !HEARTBEAT_NAMESPACE_PATTERN.matcher(
```

Comment:
This filter will be used in all `LoadSheddingStrategy`, I think we can add a method like `getBundleDataForLoadShedding` in LoadData.

Reply:
+1